docs/defi/chain-key-tokens/ckbtc/overview.mdx

```diff
@@ -71,14 +74,22 @@ Learn more in the [Bitcoin integration technical overview](/docs/references/bitc
 
 - [Chain-key tokens](https://learn.internetcomputer.org/hc/en-us/articles/34211397080980-Chain-Key-Tokens).
 
-- [ckBTC](https://github.com/dfinity/ic-js/tree/main/packages/ckbtc) JavaScript library.
-
 - [GitHub repository](https://github.com/dfinity/ic/tree/master/rs/bitcoin/ckbtc/minter).
 
-- [Local development workflow](/docs/build-on-btc/using-regtest).
+- [ckBTC overview](https://dashboard.internetcomputer.org/bitcoin) on the ICP Dashboard.
+
+- [ckTESTBTC overview](https://dashboard.internetcomputer.org/testbtc) on the ICP Dashboard.
+
+- [ckbtc](https://github.com/dfinity/ic-js/tree/main/packages/ckbtc) JavaScript library for interfacing with the ckBTC minter.
+
+- [ledger-icrc](https://github.com/dfinity/ic-js/tree/main/packages/ledger-icrc) JavaScript library for interfacing with the ckBTC ledger.
+
+- [icrc-ledger-agent](https://crates.io/crates/icrc-ledger-agent)
 
 - [Developer Liftoff: ckBTC and Bitcoin integration](/docs/tutorials/developer-liftoff/level-4/4.3-ckbtc-and-bitcoin).
 
 - [Deploying your first Bitcoin dapp](https://github.com/dfinity/examples/tree/master/motoko/basic_bitcoin).
 
-- [Creating a ckBTC point of sale dapp](https://github.com/dfinity/examples/tree/master/motoko/ic-pos).
\ No newline at end of file
+- [Creating a ckBTC point of sale dapp](https://github.com/dfinity/examples/tree/master/motoko/ic-pos).
+
+- [All about ckBTC](https://www.youtube.com/watch?v=cBZlTX818L8) DFINITY Twitter Space from February 2023.
```

Comment:
we could add those as well

https://youtu.be/FKvpar9U6wM?si=l5kU-aXSc5gFZPDL&t=2869
https://youtu.be/TwOEEBtNjzU?si=DAYAeDj4FxrGQX7E&t=2856 (more recent)